dex/src/main/scala/com/wavesplatform/dex/db/AssetsDb.scala

```diff
@@ -0,0 +1,38 @@
+package com.wavesplatform.dex.db
+
+import cats.Applicative
+import cats.syntax.applicative._
+import cats.syntax.functor._
+import com.wavesplatform.dex.db.leveldb.LevelDb
+import com.wavesplatform.dex.domain.asset.Asset
+import com.wavesplatform.dex.domain.asset.Asset.IssuedAsset
+import com.wavesplatform.dex.grpc.integration.dto.BriefAssetDescription
+
+trait AssetsDb[F[_]] {
+  def put(asset: IssuedAsset, item: BriefAssetDescription): F[Unit]
+  def get(asset: IssuedAsset): F[Option[BriefAssetDescription]]
+}
+
+object AssetsDb {
+
+  def levelDb[F[_]](levelDb: LevelDb[F]): AssetsDb[F] = new AssetsDb[F] {
+    override def put(asset: IssuedAsset, record: BriefAssetDescription): F[Unit] = levelDb.readWrite(_.put(DbKeys.asset(asset), Some(record)))
+    override def get(asset: IssuedAsset): F[Option[BriefAssetDescription]] = levelDb.readOnly(_.get(DbKeys.asset(asset)))
+  }
+
+  implicit final class Ops[F[_]: Applicative](val self: AssetsDb[F]) {
```

Comment:
1. consider name ```AssetsDbOps```
2. consider reengineering:

```
implicit final class Ops[F[_]: Applicative](val self: AssetsDb[F])
```

to

```
implicit final class AssetsDbOps[F[_]](val self: AssetsDb[F]) extends AnyVal {
  def contains(asset: IssuedAsset)(implicit F: Applicative[F]): F[Boolean] = ...
}
```

Reply:
Cool, I haven't known that we can reuse the same name (F)